usage/conf/usage.conf

```diff
@@ -0,0 +1,28 @@
+env USER=media-service
+env USER_HOME=/home/media-service
+env JAR=/home/media-service/app.jar
+
+# Workaround for https://github.com/aws/aws-sdk-java/issues/123
+env JAVA_OPTS="-XX:MaxRAMFraction=2 -XX:InitialRAMFraction=2 -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:gc.log"
```

Comment:
Borrowed this from one of the Ophan services. It sets the RAM usage for the JVM as a fraction rather than a hard coded value.

I'm not sure of the consequence of removing the default file encoding here, so I suspect we _should_ be setting it.